I do magic/promises-callbacks-js/src/js/load-data.js

```diff
@@ -1,10 +1,22 @@
-function validateEnter(e) {
-  if (e.keyCode===13) getAgents('https://api.myjson.com/bins/uptto');
-}
+const ENTER_KEY = 13;
+const URL_TO_RETIREVE_DATA = 'https://api.myjson.com/bins/uptto';
+
+// Event handlers
+window.addEventListener('load', function() {
+  getAgents(URL_TO_RETIREVE_DATA);
+});
+
+document.getElementById('lensButton').onclick = function() {
+  getAgents(URL_TO_RETIREVE_DATA);
+};
+
+document.getElementById('searchText').onkeypress = (event) => {
+  if (event.keyCode===ENTER_KEY) getAgents(URL_TO_RETIREVE_DATA);
```

Comment:
Use spaces between operators. And prefer to use 
```
if (condition) {
    expression
}
```
than `if (condition) expression`. Or at least `if (condition) { expression }`.

Reply:
Ack